app/Resources/translations/default/AdminNotificationsError.xlf

```diff
@@ -1420,47 +1529,65 @@ Comment: check if some ids are in list_skip_actions and forbid deletion</note>
-        <note>Line: 1762</note>
+        <note>Line: 1938</note>
       </trans-unit>
       <trans-unit id="546710db975c663e306008fda9fc5865">
         <source>Invalid action: this order has not been delivered.</source>
         <target>Invalid action: this order has not been delivered.</target>
-        <note>Line: 1771</note>
+        <note>Line: 1947</note>
       </trans-unit>
       <trans-unit id="b5ab864b320f4fcbcd061ee3119cf21c">
         <source>Invalid action: this order has already been delivered.</source>
         <target>Invalid action: this order has already been delivered.</target>
-        <note>Line: 1775</note>
+        <note>Line: 1951</note>
       </trans-unit>
       <trans-unit id="3e19a894b9552d54a62e99b3f7e7cf94">
         <source>Invalid action: this order has already been paid.</source>
         <target>Invalid action: this order has already been paid.</target>
-        <note>Line: 1767</note>
+        <note>Line: 1943</note>
       </trans-unit>
       <trans-unit id="7621aa68f3fb11dce889ba85f3a74758">
         <source>Invalid action: this order has not been paid.</source>
         <target>Invalid action: this order has not been paid.</target>
-        <note>Line: 1779</note>
+        <note>Line: 1955</note>
       </trans-unit>
       <trans-unit id="b19e2fa860ab1d9fdd60f5b7772b6a3e">
         <source>The product customization picture could not be found.</source>
         <target>The product customization picture could not be found.</target>
-        <note>Line: 1590</note>
+        <note>Line: 1718</note>
       </trans-unit>
       <trans-unit id="074ed474a7e0b671a07422f3014dadfb">
         <source>This product is already in your order, please edit the quantity instead.</source>
         <target>This product is already in your order, please edit the quantity instead.</target>
-        <note>Line: 1799</note>
+        <note>Line: 1975</note>
       </trans-unit>
       <trans-unit id="5b5d79a83148583db9a987bc0dbb6265">
         <source>This product is already in the invoice [1], please edit the quantity instead.</source>
         <target>This product is already in the invoice [1], please edit the quantity instead.</target>
-        <note>Line: 1803</note>
+        <note>Line: 1979</note>
       </trans-unit>
       <trans-unit id="963830daedc95e3bd008090ae4c7d1d9">
         <source>You cannot edit the price of a product that no longer exists in your catalog.</source>
         <target>You cannot edit the price of a product that no longer exists in your catalog.</target>
-        <note>Line: 1808</note>
+        <note>Line: 1984</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="src/PrestaShopBundle/Form/Admin/Catalog/Category/AbstractCategoryType.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="fc42865ab309f2e642480630b92db575">
+        <source>This field cannot be empty</source>
```

Comment:
Are you sure? Since it's an "inline" error message displayed next to the input And it's used in many other places, should we fix them all? I still have trouble understanding/remembering when/where we need the period or not 😅

![Capture d’écran 2021-05-20 à 10 15 43](https://user-images.githubusercontent.com/13801017/118943904-5a66a580-b954-11eb-8ccb-466f3a871cab.png)